test/com/facebook/buck/android/AndroidBinaryIntegrationTest.java

```diff
@@ -347,8 +347,22 @@ public void testLibraryMetadataChecksum() throws IOException {
     }
     file.close();
     contents.close();
+  }
+
+  @Test
+  public void testApkEmptyResDirectoriesBuildsCorrectly() throws IOException {
+    workspace.runBuckCommand("build", "//apps/sample:app_no_res").assertSuccess();
+    BuckBuildLog buildLog = workspace.getBuildLog();
 
+    buildLog.assertTargetBuiltLocally("//apps/sample:app_no_res");
```

Comment:
Since this is a fresh build, this is bound to be true. This isn't testing anything. Please remove.